src/plays/tube2tunes/Tube2Tunes.js

```diff
@@ -7,7 +7,7 @@ import Loader from './components/Loader';
 import TubeTunesLogo from './components/tubeTuneLogo.svg';
```

Comment:
@niteshseram Can you please revert this change. Its fixed separately, now this change will cause the build fail

Reply:
Reverted the change @atapas . Thanks